pkg/adapter/pubsub.go

```diff
@@ -0,0 +1,62 @@
+/*
+Copyright 2019 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package gcppubsub
+
+import (
+	"time"
+
+	// Imports the Google Cloud Pub/Sub client package.
+	"cloud.google.com/go/pubsub"
+)
+
+type PubSubMessage interface {
```

Comment:
Is this the equivalent of https://github.com/knative/eventing-sources/blob/master/contrib/gcppubsub/pkg/reconciler/pubsub_wrapper.go ?

If so, why not copy the 'mature' one from https://github.com/knative/eventing/blob/master/contrib/gcppubsub/pkg/util/pubsub_wrapper.go ?